modules/nw-networkpolicy-create-cli.adoc

```diff
@@ -116,6 +117,127 @@ spec:
 ifdef::multi[]
 +
 --
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+.Allow ingress traffic to one pod from a particular namespace
++
+This policy allows traffic to pods labelled `a` from pods running in `namespace-y`.
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: allow-traffic-pod
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector:
+   matchLabels:
+      pod: a
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+           kubernetes.io/metadata.name: namespace-y
+----
+ifdef::multi[]
++
+--
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+ifdef::multi[]
+.Restrict traffic to a service
++
+This policy blocks traffic to pods without the `app=bookstore` label. Traffic to pods with the `app=bookstore` label is allowed. Your application could be a REST API server, marked with labels `app=bookstore` and `role=api`. 
++
+The use cases this example addresses are: 
+
+* Restrict traffic to a service only to other microservices that need to use it.
+* Restrict connections to a database only to the application using it.
```

Comment:
With the list lead-in as a complete sentence, the current phrasing on these bullets throws me a little.

```suggestion
* Restricting the traffic to a server to only permit the microservices that need to use it.
* Restricting the connections to a database to only permit the application using it.
```
I am on the fence about my suggestion, honestly, but I hope it opens up a line of thinking.